### PR TITLE
Added ancestors and method to custom actions

### PR DESCRIPTION
**Pull request by MarcReniu**
Added more logic to custom actions.

**method**: for `post`, `destroy` or other actions, enabling 
**ancestors**: polymorphic_url needs records to perform nesting paths. This new parameter adds the logic to search for record's related objects.

### CODE REVIEW

File: app/helpers/ct_table_for/application_helper.rb [hunk 1]
```diff
@@ -215,11 +215,19 @@ def class_for_action action, options
 
     def button_for_custom_action record, options, extras
       parsed_extras = parse_extras(extras)
-      if options[:actions][:icons] != false
+      if parsed_extras[:icons] != false
```
Comment (tramuntanal) [inline app/helpers/ct_table_for/application_helper.rb]: May `if parsed_extras[:icons]` be enough?

maybe `if parsed_extras[:icons].present?`

Comment (MarcReniu) [reply app/helpers/ct_table_for/application_helper.rb]: I updated this logic. It had a bug when `icons` was `"false"`

**Comment by agustibr** [comment]
@MarcReniu since merging #13 there's a conflict in `docs/index.html` (translated to English). :smile_cat:

**Comment by MarcReniu** [comment]
Updated docs!

**Pull request merged**